### ISSUE TITLE
Dark Gtk-theme

### ISSUE THREAD

**Issue by hanny00** [flathub/de.willuhn.Jameica#2]
I want to change the gtk-theme from Jameica-Hibiscus Homebanking.
I have read that i can do it with `flatpak run --env=GTK_THEME=Breath-Dark de.willuhn.Jameica` but it does not work.

**Comment by hanny00** [comment]
I found a solution
```
1. flatpak search breeze-dark
2. install it for user
3. flatpak run --env=GTK_THEME=Breeze-Dark de.willuhn.Jameica and flatpak override --user --env=GTK_THEME=Breeze-Dark 

```

**Comment by tobias-hammerschmidt** [comment]
I think this is not specific for this application but rather to any application distributed using flatpak. Theming support should be available from flatpak 0.8.4 (https://blog.tingping.se/2017/05/11/flatpak-theming.html) and should also work with system installed themes (https://www.linuxuprising.com/2018/05/how-to-get-flatpak-apps-to-use-correct.html). Anyway good that you found a solution which works for you.
